src/config-hardware/fix-audio.md

```diff
@@ -0,0 +1,77 @@
+# Useful Audio Commands
+
+Use Terminal commands to restart daemons and drivers that control audio streams.
+
+---
+
+## ALSA Commands
+
+ALSA commands can be used to view audio-related diagnostic information and to reset audio drivers.
+
+View all devices detected by ALSA:
+
+```bash
+aplay -l
+```
+
+View all recording devices detected by ALSA:
+
+```bash
+arecord -l
+```
+
+Reload sound driver modules: 
+
+```bash 
+sudo alsa force-reload
+```
+
+The `alsa-info` command gathers a number of outputs, including some of the above-listed outputs, and packages them for convenient sharing.
+
+```bash
+alsa-info
+```
+
+## Additional Audio Commands
+
+### List Detected Sound Cards
+
+If ALSA doesn't list a sound card, it may not be physically detected by the system at all. If the Linux kernel sees a sound card, it will show up in your `lspci` output. This command will list every sound card your system detects, and show the driver being used for each one:
+
+```bash
+lspci -v | grep -A6 Audio
+```
+
+### Check PipeWire's Status
+
+This command will check the status of PipeWire and show any errors if automatic restarts raised any errors:
+
+```bash
+systemctl --user status pipewire
```

Comment:
`journalctl --unit pipewire --user` (or `journalctl -u pipewire --user`) will show all of the log history (or add `-b 0` for only the current boot.)

`pipewire` (which actually defaults to `pipewire.service`) is not the only PipeWire-related service. There is also:

- pipewire.socket
- pipewire-session-manager.service
- pipewire-media-session.service
- pipewire-pulse.service
- pipewire-pulse.socket

You can view logs for all of these at once with a command like this:

```
journalctl -u pipewire.service -u pipewire.socket -u pipewire-session-manager.service -u pipewire-media-session.service -u pipewire-pulse.service -u pipewire-pulse.socket --user
```

(Again, adding `-b 0` if you only want to see the current boot to check for errors.)

Reply:
I've added this info to a new section on this page "Viewing PipeWire Logs".